contrib/IECoreAppleseed/src/IECoreAppleseed/outputDriver/DisplayTileCallback.cpp

```diff
@@ -392,7 +392,8 @@ extern "C"
 {
 
 // Display plugin entry point.
-IECOREAPPLESEED_API asr::ITileCallbackFactory* create_tile_callback_factory( const asr::ParamArray *params )
+IECOREAPPLESEED_API asr::ITileCallbackFactory* create_tile_callback_factory( const asr::ParamArray *params );
+asr::ITileCallbackFactory* create_tile_callback_factory( const asr::ParamArray *params )
```

Comment:
Does this even compile? Seems like there shouldn't be a double-up of functions and no export here as there's already an extern "C" which should have a clean exported unmangled function for Appleseed to load in.

Reply:
Yeah, it's a little confusing with the declaration and definition right there, but it's compiling here so I'm inclined to resolve it as-is.